File: Java/PC.java
public class PC {
    // untuk menampung data dari processor, disk, ram, dan total harga
    private Processor processor1;  
    private Ram ram1;     
    private Disk disk1;   
    private int totalPrice; 

    // constructor
    public PC() {
    }

    public PC(Processor processor1, Ram ram1, Disk disk1, int totalPrice) {
        this.processor1 = processor1;
        this.ram1 = ram1;
        this.disk1 = disk1;
        this.totalPrice = totalPrice;
    }

    // setter and getter
    public void setProcessor(Processor processor1)
    {
        this.processor1 = processor1;
    }

    public Processor getProcessor()
    {
        return this.processor1;
    }

    public void setDisk(Disk disk1)
    {
        this.disk1 = disk1;
    }

    public Disk getDisk()
    {
        return this.disk1;
    }

    public void setRAM(Ram ram1)
    {
        this.ram1 = ram1;
    }

    public Ram getRAM()
    {
        return this.ram1;
    }

    public void setTotalPrice(int totalPrice)
    {
        this.totalPrice = totalPrice;
    }

    public int getTotalPrice()
    {
        return this.totalPrice;
    }

    // method
    public void printPC() {
        System.out.println("==============================");
        this.processor1.printProcessor();
        this.ram1.printRam();
        this.disk1.printDisk();
        System.out.println("Total Harga : Rp " + this.getTotalPrice());
        System.out.println("==============================");
    }
}

File: Python/PC.py
from Processor import Processor
from Ram import Ram
from Disk import Disk

class PC:
    #untuk menampung data dari processor, disk, ram, dan total harga
    __processor1 = Processor()    
    __ram1 = Ram()                
    __disk1 = Disk()            
    __totalPrice = 0                

    # constructor
    def __init__(self, Processor1, ram1, disk1, totalPrice = 0):
        self.__processor1 = Processor1
        self.__ram1 = ram1
        self.__disk1 = disk1
        self.__totalPrice = totalPrice

    # setter and getter
    def setProcessor(self, processor1):
        self.__processor1 = processor1
    
    def getProcessor(self):
        return self.__processor1

    def setRAM(self, ram1):
        self.__ram1 = ram1

    def getRAM(self):
        return self.__ram1

    def setDisk(self, disk1):
        self.__disk1 = disk1
    
    def getDisk(self):
        return self.__disk1
    
    def setTotalPrice(self, totalPrice):
        self.__totalPrice = totalPrice
    
    def getTotalPrice(self):
        return self.__totalPrice

    # method
    def printPC(self):
        print("==============================")
        self.__processor1.printProcessor()
        self.__ram1.printRAM()
        self.__disk1.printDisk()
        print("Total Harga : Rp " + str(self.getTotalPrice()))
        print("==============================")